libraries/aws_elb.rb

```diff
@@ -43,6 +43,10 @@ def initialize(opts = {})
       policies_in_use     = resp.listener_descriptions.map(&:policy_names).flatten.uniq
       @ssl_policies       = @aws.elb_client.describe_load_balancer_policies(load_balancer_name: opts[:load_balancer_name])
                                 .policy_descriptions.select { |p| policies_in_use.include?(p.policy_name) }
+      @cross_zone_load_balancing = @aws.elb_client.describe_load_balancer_attributes(load_balancer_name: opts[:load_balancer_name])
```

Comment:
@Rohit1509 , Could we please do the n/w call to `describe_load_balancer_attributes` only once and set both the attributes using the attributes received ? 

Also a couple of recommendations would be to 
* lazy load this only when the method is called
* since this returns boolean flag, please try using matchers syntax for this .. 
for example, 

you can implement methods
`cross_zone_load_balancing_enabled?` & `access_log_enabled?`
which will be available in inspec controls as 
 ```
it { should be_cross_zone_load_balancing_enabled }
it { should be_access_log_enabled }
```

Reply:
fixed